Actions fail with Python 3.12.7 but passed with 3.12.6

The error is raised during argument parsing:

```
----------------------------- Captured stderr call -----------------------------
usage: voc4cat docs [-h] [-v | -q] [--config CONFIG] [-O DIRECTORY]
                    [-l LOGFILE] [--style {p,y,l,o,d,e}] [--force]
                    VOCAB
voc4cat docs: error: argument --style: invalid choice: 'pylode' (choose from 'p', 'y', 'l', 'o', 'd', 'e')
_______________________________ test_build_docs ________________________________
/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/argparse.py:1931: in parse_known_args
    namespace, args = self._parse_known_args(args, namespace)
/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/argparse.py:2168: in _parse_known_args
    start_index = consume_optional(start_index)
/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/argparse.py:2099: in consume_optional
    take_action(action, args, option_string)
/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/argparse.py:1991: in take_action
    argument_values = self._get_values(action, argument_strings)
/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/argparse.py:2536: in _get_values
    self._check_value(action, value)
/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/argparse.py:2595: in _check_value
    raise ArgumentError(action, msg % args)
E   argparse.ArgumentError: argument --style: invalid choice: 'pylode' (choose from 'p', 'y', 'l', 'o', 'd', 'e')
```

Did argparse change? 
Yes, there were several changes: https://docs.python.org/release/3.12.7/whatsnew/changelog.html#python-3-12-7
Most likely, the error is due to stricter handling of choices: https://github.com/python/cpython/pull/124578/

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -41,7 +41,7 @@ dependencies = [
   "colorama",
   "curies >= 0.6.6",
   "networkx >= 2.8",
-  "openpyxl >= 3.0.9",
+  "openpyxl >= 3.1.5",
   "pillow >= 9.1.0",
   "pydantic < 2.0.0",
   "pyLODE < 3.0.0",

--- a/src/voc4cat/cli.py
+++ b/src/voc4cat/cli.py
@@ -386,7 +386,7 @@ def add_docs_subparser(subparsers, options):
     parser.add_argument(
         "--style",
         help="Select style of html documentation. (default: pylode)",
-        choices=("pylode"),
+        choices=("pylode",),
         default="pylode",
         type=str,
         required=False,

